--- src/ipa/Makefile.orig	2026-04-30 15:26:59
+++ src/ipa/Makefile	2026-04-30 15:23:12
@@ -183,7 +183,7 @@
 ACLOCAL = ${SHELL} '/opt/local/var/macports/build/libwmf-bc41937e/work/libwmf-0.2.15/build-aux/missing' aclocal-1.17
 AMTAR = $${TAR-tar}
 AM_CFLAGS =  -I/opt/local/include/freetype2 -I/opt/local/include/libpng16       
-AM_CPPFLAGS = -I/opt/local/include -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX26.sdk -I$(top_srcdir)/include \
+AM_CPPFLAGS = -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX26.sdk -I$(top_srcdir)/include \
 	-I$(top_srcdir)/src $(GDHDRS) -I$(top_builddir)
 AM_DEFAULT_VERBOSITY = 1
 AM_LDFLAGS =    -L/opt/local/lib -lfreetype  -lexpat -ljpeg -lpng -lz -lm
@@ -197,7 +197,7 @@
 CCDEPMODE = depmode=gcc3
 CFLAGS = -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX26.sdk -arch x86_64
 CPP = /usr/bin/clang -E
-CPPFLAGS = -I/opt/local/include -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX26.sdk
+CPPFLAGS = -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX26.sdk
 CSCOPE = cscope
 CTAGS = ctags
 CYGPATH_W = echo
